README.md

```diff
@@ -95,20 +98,42 @@ This plugin decorates fastify with a `mail` object containing the following meth
 
 - `createMessage`: `function` - Generates a message from templates with context injected. 
   - Accepts the following arguments: 
-    - `recipients`: `array` - Array containing recipient[s] email address (`string`).
-    - `templates`: `string` - the relative path to the message's templates.
-    - `context`: `object` - Object containing context for the message.
+    - `message`: `array` -  Comma separated list or an array of recipients email addresses (`string`) that will appear on the To: field
```

Comment:
[nit] could we either remove the backtick usage for keys (i don't know if that's standard, it reads off to me), and maybe just use single quotes for the value type (again doing away with strange to my eyes backtick usage)?

Reply:
done - not sure if it looks any better.